web/sites/default/settings.php

```diff
@@ -42,17 +42,28 @@
   'bower_components',
 ];
 
+// $settings['updates_log_site'] = 'my-project-name';
```

Comment:
Shouldn't there also be section in readme for getting this up and running?

Reply:
@ragnarkurmwunder, I propose we leave `updates_log_site` and `updates_log_env` out from this template as it's targeting projects in Silta and the `updates_log` features are documented in it's repository.